projects/plugins/jetpack/extensions/blocks/google-docs-embed/embed-placeholder.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+
+import { Button, Placeholder } from '@wordpress/components';
+
+const EmbedPlaceHolder = ( {
+	className,
+	icon,
+	instructions,
+	label,
+	notices,
+	url,
+	onSubmit,
+	placeholder,
+	updateUrl,
+} ) => {
+	return (
+		<div className={ classnames( 'wp-block-jetpack-google-docs-embed-editmode', className ) }>
+			<Placeholder icon={ icon } label={ label } instructions={ instructions } notices={ notices }>
+				<form onSubmit={ onSubmit }>
+					<input
+						type="url"
+						value={ url || '' }
+						className="components-placeholder__input"
+						placeholder={ placeholder }
+						onChange={ event => updateUrl( event.target.value ) }
+					/>
+					<Button isPrimary type="submit">
```

Comment:
You should be able to use `variant="primary"` in `Button` nowadays.

Reply:
This has been updated.